private/model/api/api.go

```diff
@@ -77,6 +77,8 @@ type API struct {
 
 	// Set to true to strictly enforce usage of the serviceId for the package naming
 	StrictServiceId bool
+
+	AwsQueryCompatible map[string]AwsQueryCompatibleError
```

Comment:
Nit, Go style prefers acronyms to be all upper case, i.e. `AWSQueryCompatible`, and `AWSQueryCompatibleError`